src/components/icons/GithubIcon.svelte

```diff
@@ -0,0 +1,11 @@
+<script>
+	import BaseSvg from './BaseSvg.svelte';
+</script>
+
+<BaseSvg {...$$restProps}>
```

Comment:
According to the previous comment `viewBox` should be here.

```svelte
<BaseSvg viewBox="0 0 32 32" {...$$restProps}>
````

As I try this change, the crop issue in the story is also fixed. Please change other icon too.